.github/workflows/shellcheck.yaml

```diff
@@ -24,10 +24,11 @@ jobs:
     name: Shellcheck
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-    - name: Run ShellCheck
-      uses: ludeeus/action-shellcheck@00b27aa7cb85167568cb48a3838b75f4265f2bca # master
-      with:
+      - name: checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@00b27aa7cb85167568cb48a3838b75f4265f2bca # master
+        with:
         severity: warning
```

Comment:
This change appears to be breaking the indentation in relation to `with.severtiy` etc. It also seems like an unnecessary change.

Reply:
@BlaineEXE :  I was fixing yamllint errors with this change.

Reply:
Thanks for highlighting this! Now I fixed the indentation so that shellcheck is running again.